.cloudbuild/publish-image-ecr.sh

```diff
@@ -5,7 +5,10 @@ STAGING_DOCKER_IMAGE=146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/
 PRODUCTION_DOCKER_IMAGE=public.ecr.aws/gravitational/docs:latest
 
 ## Install aws
-apt install awscli
+apt install -y unzip
```

Comment:
Why the change to AWSCLI v2?

Reply:
The `ecr-public` subcommand is not available on the default package installed by apt repositories. I've decided to just install based on aws installation guides.